src/pkg/gc/gc.go

```diff
@@ -66,8 +67,8 @@ func StartDockerClient(endpoint string) *docker.Client {
 }
 
 func DiskSpaceGC(intervalInSeconds uint64, policy GCPolicy) {
-	diskSpaceFetcher := &DiskSpaceFetcher{}
-	gocron.Every(intervalInSeconds).Seconds().Do(CleanAllWithDiskSpacePolicy, diskSpaceFetcher, policy)
+	diskSpaceFetcher = &DiskSpaceFetcher{}
```

Comment:
I would had prefered not to have, but keeping to mock in place and making this usable without having it being passed down in three methods caused this. Open to ideas how to improve, this is pretty bad